pkg/services/policer/policer.go

```diff
@@ -108,7 +108,7 @@ func defaultCfg() *cfg {
 	return &cfg{
 		log:           zap.L(),
 		batchSize:     10,
-		cacheSize:     1024, // 1024 * address size = 1024 * 64 = 64 MiB
+		cacheSize:     1 << 20, // 1024*1024 * address size = 1MiB * 64 = 64 MiB
```

Comment:
MiB? Please...

Reply:
But the biggest delay here in fact is communication. You're not likely to do more than 1000 RPS per thread, which roughly means 600000 entries for 30s

Reply:
> MiB? Please...

it was `MiB` i went `MiB`, fixed

> 600000 entries for 30s

but 30s maybe not 30s. and 1000 RPS may be 100 RPS. not sure we need this cache in general but think that 1024 is senseless for sure. this PR allows us to check it in tests. what is your concern/suggestion?

Reply:
Let's try, OK.

Reply:
my main question, in fact, is about memory usage. what do you think about 64mb for replication cache storage?

Reply:
I'm not sure we need it, but it doesn't seem to be a big problem either, if you have 1M objects it's not a small installation.